src/AltinnCore/Designer/package-lock.json

```diff
@@ -1205,7 +1205,7 @@
         "debug": {
           "version": "3.2.6",
           "resolved": "https://registry.npmjs.org/debug/-/debug-3.2.6.tgz",
-          "integrity": "sha512-mel+jf7nrtEl5Pn1Qx46zARXKDpBbvzezse7p7LqINmdoIk8PYP5SySaxEmYv6TZ0JyEKA1hsCId6DIhgITtWQ==",
```

Comment:
Why is the package-lock files changed?

Reply:
I deleted them and installed them again.
Got loads of errors on typings. Fixed it for me.